recipe/meta.yaml

```
@@ -1,14 +1,14 @@
 {% set python_min = "3.10" %}
```

```suggestion
{% set python_min = "3.11" %}
```
To sync with the source.